businesscentral-indexing/container.yaml

```diff
@@ -4,3 +4,6 @@ platforms:
   - x86_64
 compose:
   pulp_repos: true
+  include_unpublished_pulp_repos: true
+  packages:
+  - tzdata-java
```

Comment:
The changes looks good to me, but I am curious about these 3 lines, it is something now required for ubi 8 or is just a workaround until the final ubi image release?

Reply:
@spolti Ken said I can get rid of the tz-data one now, and should probably be able to get rid of the include_unpublished_pulp_repos one too, since the repos have been released. I'll give it a shot.

Reply:
great.